atest/testdata/standard_libraries/remote/libraryinfo.py

```diff
@@ -0,0 +1,38 @@
+import sys
+
+from remoteserver import RemoteServer, keyword
+
+
+class BulkLoadRemoteServer(RemoteServer):
+
+    def _register_functions(self):
+        """
+        Individual get_keyword_* methods are not registered.
+        This removes the fall back scenario should get_library_information fail.
+        """
+        self.register_function(self.get_library_information)
+        self.register_function(self.run_keyword)
+
+    def get_library_information(self):
+        info_dict = dict()
+        for kw in self.get_keyword_names():
+            kw_dict = dict()
```

Comment:
what about just:

```
info_dict[kw] = {
    'args': self.get_keyword_arguments(kw),
    'tags': self.get_keyword_tags(kw)
    'doc': self.get_keyword_documentation(kw)
}
```

Reply:
Hi @bhirsz and thanks for reviewing!

Good suggestion, I might even prefer:
```
info_dict[kw] = dict(args=self.get_keyword_arguments(kw),
                     tags=self.get_keyword_tags(kw),
                     doc=self.get_keyword_documentation(kw))
```
I have similar code in the [Python remote server implementation](https://github.com/robotframework/PythonRemoteServer/pull/74). I will update there as well.